test/dlc_tests/ec2/pytorch/training/test_pytorch_training_2_1.py

```diff
@@ -49,6 +49,36 @@ def test_pytorch_2_1_gpu(
     test_utils.execute_serial_test_cases(test_cases, test_description="PT 2.1 GPU")
 
 
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.integration("pytorch_gpu_heavy_tests")
+@pytest.mark.model("N/A")
+@pytest.mark.team("conda")
+@pytest.mark.parametrize(
```

Comment:
note: this silently removes p4de testing for TE in PT 2.1. However, given that we are already running on other heavy instance types, this should suffice